index.html

```diff
@@ -260,6 +296,55 @@ <h3>emailhere@email.com</h3>
         </div>
 
     </section>
+
+    <section class="chat-container">
+        <div class="chat-content">
+            <div class="chat-header">
+                <h1>Let's Chat</h1>
+            </div>
+            <div class="chat-desc">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I would love to help with it </p>
+            </div>
+            <form action="https://formspree.io/f/1" method="post" class="contact-form">
+                <div class="contact-inbox">
+                    <input type="text" required name="username" maxlength="30" placeholder="Full Name" >
+                </div>
+                
+                <div class="contact-inbox">
+                    <input type="email" required name="email" placeholder="Email Address" >
+                </div>
+    
+                <div class="contact-txt-area">
+                    <textarea name="desc" required maxlength="500" placeholder="Enter your message here"></textarea>
+                </div>
+                
+                <div class="btn-send">
+                    <input type="submit" value="Send">
+                </div>
+            </form>
+        
+        </div>
+
+    </section>
+
+    <div class="desk-footer">
+        <footer>
+            <div class="last-break">
+                <img src="./img/hor_line.png" alt="Horizontal Line" id="last-line-id-1">
+            </div>
+    
+            <div class="bottom-icons">
+                <ul class="social-icon" >
+                    <li><a href="#"><img src="./img/github.png" alt="Github"></a>
+                    <li><a href="#"><img src="./img/linkdin.png" alt="Linkding"></a>
+                    <li><a href="#"><img src="./img/icon-3.png" alt="Not known"></a>
+                    <li><a href="#"><img src="./img/twitter.png" alt="twitter"></a>
+                    <li><a href="#"><img src="./img/icon-4.png" alt="Not known"></a></li>
+                    <li><a href="#" id="email-contact">emailhere@email.com</a></li>
```

Comment:
- [ ] **Required** You have done a great job so far 👏. However, there are a few adjustments needed to move to the next project.

Kindly add padding to your social icons to push them in to look similar to your chosen template. 
Your Design | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150638342-8c8cb1ee-bc26-47ec-ae34-07fc0609a3b6.png) | ![image](https://user-images.githubusercontent.com/27709832/150638245-ec12670a-621b-4c86-8997-8c384c4b401a.png)